diff --git a/node_modules/react-native-custom-qr-codes-expo/lib/QRCode.js b/node_modules/react-native-custom-qr-codes-expo/lib/QRCode.js
index c8f2004..d2869a2 100644
--- a/node_modules/react-native-custom-qr-codes-expo/lib/QRCode.js
+++ b/node_modules/react-native-custom-qr-codes-expo/lib/QRCode.js
@@ -32,8 +32,8 @@ export default class QRCode extends PureComponent {
     innerEyeStyle: PropTypes.oneOf(['square', 'circle', 'diamond']),
     outerEyeStyle: PropTypes.oneOf(['square', 'circle', 'diamond']),
     codeStyle: PropTypes.oneOf(['square', 'circle', 'diamond', 'dot', 'ninja', 'sharp']),
-    logo: Image.propTypes.source,
-    backgroundImage: Image.propTypes.source,
+    logo: PropTypes.oneOfType([PropTypes.number, PropTypes.object]),
+    backgroundImage: PropTypes.oneOfType([PropTypes.number, PropTypes.object]),
     logoSize: PropTypes.number,
     ecl: PropTypes.oneOf(['L', 'M', 'Q', 'H'])
   };
